<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-17T18:07:56 IST" name="LeadAngelV1" finished-at="2024-09-17T18:08:15 IST" duration-ms="18695">
    <groups>
    </groups>
    <test started-at="2024-09-17T18:07:56 IST" name="LA_chrome" finished-at="2024-09-17T18:08:15 IST" duration-ms="18695">
      <class name="com.leadAngel.testCases.TC_Sanity_SRandGroupMapping_LeadManagement">
        <test-method is-config="true" signature="setup(java.lang.String, java.lang.String)[pri:0, instance:com.leadAngel.testCases.TC_Sanity_SRandGroupMapping_LeadManagement@109f5dd8]" started-at="2024-09-17T18:07:56 IST" name="setup" finished-at="2024-09-17T18:08:04 IST" duration-ms="7618" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="LeadManagement()[pri:0, instance:com.leadAngel.testCases.TC_Sanity_SRandGroupMapping_LeadManagement@109f5dd8]" started-at="2024-09-17T18:08:04 IST" name="LeadManagement" finished-at="2024-09-17T18:08:15 IST" duration-ms="10424" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.leadAngel.testCases.TC_Sanity_SRandGroupMapping_LeadManagement.LeadManagement(TC_Sanity_SRandGroupMapping_LeadManagement.java:54)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LeadManagement -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.leadAngel.testCases.TC_Sanity_SRandGroupMapping_LeadManagement@109f5dd8]" started-at="2024-09-17T18:08:15 IST" name="tearDown" finished-at="2024-09-17T18:08:15 IST" duration-ms="335" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.leadAngel.testCases.TC_Sanity_SRandGroupMapping_LeadManagement -->
    </test> <!-- LA_chrome -->
  </suite> <!-- LeadAngelV1 -->
</testng-results>
